index.bs

```diff
@@ -341,11 +343,11 @@ The <dfn method for="StorageBucket">persisted()</dfn> method steps are:
 
 1. If |bucket|'s [=storage bucket/removed=] flag is true, then [=queue a storage task=] to [=reject=] |p| with an {{InvalidStateError}}.
 
-1. Otherwise,
+1. Otherwise, run these steps [=in parallel=]:
 
-    1. If |bucket|'s [=bucket mode=] is `"persistent"`, then [=queue a storage task=] to [=/resolve=] |p| with true.
+    1. Let |persisted| be true if |bucket|'s [=bucket mode=] is `"persistent"`, otherwise false.
```

Comment:
nit: slightly confusing to reuse a symbol name (`persisted()` vs `persisted`)

Reply:
Lacking variable name creativity this morning, hope `persistent` is ok 😬